src/bots/meets/src/bot.ts

```diff
@@ -249,31 +271,45 @@ export class MeetingBot {
             });
         }
 
-        // Define Exit Condition(s)
-        const onlyMeInMeeting = this.page
-            .locator(onePersonRemainingField)
-            .waitFor({ timeout: 0 });
-        // const exitCondition2 = this.page.waitForFunction(() => document.title.includes('Success'));
-
-        // Chill Until an Exit Condition is met
-        const result = await Promise.race([
-            onlyMeInMeeting,
-            // Add more conditions here later
-        ]);
+        // Loop -- check for end meeting conditions every second
+        console.log('Waiting until a leave condition is fulfilled..')
+        while (true) {
+
+            // Check if it's only me in the meeting
+            if (await this.page.locator(onePersonRemainingField).count() > 0) break;
+            
+            // Got kicked
+            if (await this.page.locator(gotKickedDetector).count() > 0) {
+                this.kicked = true;
+                console.log('Kicked');
+                break;
+            }
+            
+            // Reset Loop
+            setTimeout(1000); //1 second loop
```

Comment:
This needs to be awaited

```suggestion
            await setTimeout(1000); //1 second loop
```